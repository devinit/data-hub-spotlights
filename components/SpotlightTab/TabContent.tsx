import React, { FunctionComponent } from 'react';

const TabContent: FunctionComponent = ({ children }) => {
  return (
    <article className="tabs__content">
      { children }
    </article>
  );
};

export { TabContent };
