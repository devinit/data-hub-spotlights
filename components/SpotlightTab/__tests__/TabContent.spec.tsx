/**
 * @jest-environment jsdom
 */
import '@testing-library/jest-dom/extend-expect';
import React from 'react';
import * as TestRenderer from 'react-test-renderer';
import { TabContent } from '../TabContent';

describe('TabContent', () => {
  test('renders correctly', () => {
    const renderer = TestRenderer.create(<TabContent />).toJSON();

    expect(renderer).toMatchSnapshot();
  });

  test('renders with children', () => {
    const renderer = TestRenderer.create(
      <TabContent>
        <div>My Child</div>
      </TabContent>
    ).toJSON();

    expect(renderer).toMatchSnapshot();
  });
});
