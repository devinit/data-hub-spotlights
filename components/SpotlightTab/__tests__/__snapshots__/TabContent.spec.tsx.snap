// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabContent renders correctly 1`] = `
<article
  className="tabs__content"
/>
`;

exports[`TabContent renders with children 1`] = `
<article
  className="tabs__content"
>
  <div>
    My Child
  </div>
</article>
`;
